core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -7072,6 +7144,175 @@ components:
                 type: string
               message:
                 type: string
+############################
+# REQUEST: /stream/proofs #
+###########################
+    StreamProofsRequest:
+      description: |
+        A request to retrieve a sublist of proofs.
+      type: object
+      required:
+        - network
+        - filter
+      properties:
+        network:
+          $ref: "#/components/schemas/NetworkIdentifier"
+        filter:
+          $ref: "#/components/schemas/StreamProofsFilter"
+        max_page_size:
+          description: If specified, the maximum number of proofs that will be returned.
+          type: integer
+        continuation_token:
+          $ref: "#/components/schemas/ContinuationToken"
+    StreamProofsFilterType:
+      type: string
+      enum:
+        - All
+        - NewEpochs
+        - ProtocolUpdateTriggers
+        - ProtocolUpdateExecution
+    StreamProofsFilter:
+      type: object
+      required:
+        - type
+      properties:
+        type:
+          $ref: "#/components/schemas/StreamProofsFilterType"
+      discriminator:
+        propertyName: type
+        mapping:
+          # NOTE: These need to match StreamProofsFilterType
```

Comment:
`ProtocolUpdateInitializations` doesn't match (StreamProofsFilterType is `ProtocolUpdateTriggers` )

Reply:
Good spot!